application-changerequest-ui/src/main/resources/ChangeRequest/Code/ChangeRequestTranslations.ja.xml

```diff
@@ -137,7 +137,7 @@ ChangeRequest.Code.ConfigurationClass_mergeUser=ユーザーをマージ
 ### Missing: ChangeRequest.Code.ConfigurationClass_delegateEnabled=Enable delegate approvers
 ### Missing: ChangeRequest.Code.ConfigurationClass_delegateEnabled.hint=If enabled, allow to use delegate approval mechanism: the delegates are users allowed to perform reviews on behalf of the official explicit approvers of a page. 
 ### Missing: ChangeRequest.Code.ConfigurationClass_delegateClassPropertyList=User properties to use for computing delegates
-### Missing: ChangeRequest.Code.ConfigurationClass_delegateClassPropertyList.hint=This configuration specifies the list of XWikiUsers properties that should be used for automatically compute the delegate approvers. If the list is empty then the delegates should be set manually.
+### Missing: ChangeRequest.Code.ConfigurationClass_delegateClassPropertyList.hint=This configuration specifies the list of XWikiUsers properties that should be used to automatically compute the delegate approvers. If the list is empty then the delegates should be set manually.
```

Comment:
You don't need to edit this as it's a missing translation (see the `### Missing:` prefix), meaning it's not used. Just a placeholder for avoiding problem with the file generation.

Reply:
Now it's harmless so merging like this.